### PR TITLE
[bvl_feedback] Update TestPlan

### LINKED ISSUE #7031: [bvl_feedback] Update Testplan
The current test plan should provide more detailed information on where to find instructions and feedback windows.

### PR DESCRIPTION
**Pull request by haoweiqiu**
## Brief summary of changes

- [x] Have you updated related documentation?

Updated BVL_feedback testplan.

Resolved #7031

### CODE REVIEW

**Comment by laemtl** [comment]
Thanks @haoweiqiu, can you create an issue linked to this PR and publish it?

**Review by laemtl — APPROVED** [review]

**Review by maltheism — APPROVED** [review]

File: modules/bvl_feedback/test/TestPlan.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 # Behavioural Feedback Test Plan
 
-1. Behavioural feedback button (notepad in the toolbar) should show up on the following pages:
- * Candidate Profile
- * Instrument List
- * Any instrument
-[Automation Test]
+1. Behavioural feedback button should show up on the following pages:
+ * Candidate Profile  
+ To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.
```
Comment (christinerogers) [inline modules/bvl_feedback/test/TestPlan.md:5]: This edit is not necessarily unless you'd like to change it to tell the tester _where_ to look for feedback within each of these modules/pages (as with the pencil icon).  
It is very safe to assume that anyone testing Loris knows how to get to a candidate profile and instrument list page.  I see a few inaccuracies below, i'll correct as I go ...

File: modules/bvl_feedback/test/TestPlan.md [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
- * Any instrument
-[Automation Test]
+1. Behavioural feedback button should show up on the following pages:
+ * Candidate Profile  
+ To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.  
+ ![image](images/candidate_list.PNG)  
+ * Instrument List  
+ To find an instrument, first navigate to a specific Candidate Profile, then click on a specific Visit Label.  
+ ![image](images/visit_label.PNG)  
+ A list of Intruments shall be found in this page as shown below.
```
Comment (christinerogers) [inline modules/bvl_feedback/test/TestPlan.md:10]:
```suggestion
 A list of Instruments is found in this page as shown below.  
```
typo and changing the tone - @haoweiqiu you'll notice we've basically banned "should" and especially "shall" from all our docs - that's a conscious decision for writing style.

File: modules/bvl_feedback/test/TestPlan.md [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
- * Candidate Profile
- * Instrument List
- * Any instrument
-[Automation Test]
+1. Behavioural feedback button should show up on the following pages:
+ * Candidate Profile  
+ To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.  
+ ![image](images/candidate_list.PNG)  
+ * Instrument List  
+ To find an instrument, first navigate to a specific Candidate Profile, then click on a specific Visit Label.
```
Comment (christinerogers) [inline modules/bvl_feedback/test/TestPlan.md:8]:
```suggestion
Feedback on a whole Visit is accessed via the Instrument List page - click the pencil icon in the blue menu bar.
```
Important - for visit-level feedback, use the pencil icon while you're looking at the instrument list module (don't click through to a visit label)

File: modules/bvl_feedback/test/TestPlan.md [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
+ * Candidate Profile  
+ To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.  
+ ![image](images/candidate_list.PNG)  
+ * Instrument List  
+ To find an instrument, first navigate to a specific Candidate Profile, then click on a specific Visit Label.  
+ ![image](images/visit_label.PNG)  
+ A list of Intruments shall be found in this page as shown below.  
+ ![image](images/instrument_list.PNG)  
+ * Any instrument [Automation Test]   
+ *Note: The feedback button has the notepad icon in the toolbar as shown below.*
```
Comment (christinerogers) [inline modules/bvl_feedback/test/TestPlan.md:13]: across loris 
```suggestion
 *Note: The feedback button is represented by the notepad icon in the menu as shown below.*  
```
* Menu (not toolbar)

Comment (christinerogers) [reply modules/bvl_feedback/test/TestPlan.md:13]: i think you should move this to the first line of your modifications - this pencil/notepad icon is the same trigger across multiple modules, not just in the instrument form

**Review by christinerogers — CHANGES_REQUESTED** [review]
some initial comments (see inline below) and typo found. 
Note how we describe elements consistently across loris documentation, e.g. menu (not a toolbar), pencil icon (not notepad) - important.

**Comment by christinerogers** [comment]
Hi @haoweiqiu Could you address these soon?

**Comment by laemtl** [comment]
@haoweiqiu Try to rebase to see if this helps passing the tests.

**Review by christinerogers — APPROVED** [review]

**Comment by christinerogers** [comment]
This could be reviewed ahead of final merge, @driusan.

**Comment by ridz1208** [comment]
@cmadjar what's the discussion about ?

**Comment by cmadjar** [comment]
@ridz1208 can you rebase?

**Comment by cmadjar** [comment]
@driusan ready for you :)

**Comment by ridz1208** [comment]
@driusan why was discussion required re-added on this ?

**Review by driusan — CHANGES_REQUESTED** [review]
I don't think this test plan warrants including binary image files. Git is not a good place to track binary files, and they are not useful when reading the testplan outside of github. I don't see any of the usages here being really required.

**Comment by ridz1208** [comment]
Similar changes made causing conflicts

**Pull request closed without merging**